package-lock.json

```diff
@@ -0,0 +1,5187 @@
+{
```

Comment:
remove this `package-lock.json` file - we're using yarn internally in the library